### PR TITLE
Load testing

### PR DESCRIPTION
**Pull request by tomas-stefano**
[Trello](https://trello.com/c/GFvYFTdR/1033-performance-tests-for-api-metadata)

## Context

This is the load testing script for the metadata api. 

The main idea is to run this every time we need to have a sense of idea for each endpoint of the app.

Obs.: This script is agnostic of what language and platform we are using for the metadata api.

### CODE REVIEW

File: app/serialisers/errors_serializer.rb [hunk 1]
```diff
@@ -6,6 +6,7 @@ def initialize(message:)
   end
 
   def attributes
+    Rails.logger.info(message)
```
Comment (brenetic) [inline app/serialisers/errors_serializer.rb:9]: are we keeping this in?

File: bin/load-test [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+  echo "GET $get_specific_version_url" | \
+  vegeta attack -rate=$rate \
+  -header "Content-type: application/json" \
+  -header "Authorization: Bearer ${access_token}" \
+  -header "Accept: application/json" \
+  -duration=$duration | \
+  tee results.bin | \
+  vegeta report
+
+## GET USER ##
```
Comment (brenetic) [inline bin/load-test:82]:
```suggestion
## GET USER SERVICES##
```

File: Makefile [hunk 3]
```diff
@@ -20,9 +20,13 @@ unit: build
 
 .PHONY: seed_public_key
 seed_public_key:
-	$(DOCKER_COMPOSE) up -d --build metatada-app-service-token-cache-redis
+	$(DOCKER_COMPOSE) up -d --build metadata-app-service-token-cache-redis
```
Comment (brenetic) [inline Makefile:23]: 😆

File: app/serialisers/errors_serializer.rb (hunk 1 above)
Comment (tomas-stefano) [reply app/serialisers/errors_serializer.rb:9]: Better not. I was considering leaving a Rails.logger.debug there to be only on dev environment but I will remove.

**Review by brenetic — APPROVED** [review]
🚀

**Pull request merged**